syft/generic/pointers/pointer_tensor.py

```diff
@@ -281,12 +281,6 @@ def move(self, destination: AbstractWorker, requires_grad: bool = False):
 
         ptr = self.remote_send(destination, requires_grad=requires_grad)
 
-        # We make the pointer point at the remote value. As the id doesn't change,
-        # we don't update ptr.id_at_location. See issue #3217 about this.
-        # Note that you have now 2 pointers on different locations pointing to the
-        # same tensor.
-        ptr.location = destination
-
         return ptr
```

Comment:
Maybe we already answered this question but what is the difference between remote_send and move now?
Because I see written
```
def move():
     return self.remote_send()
```

Reply:
There isn't much difference actually, except the one you just mention: it's the same remote operation but you get as an answer not the same pointer